pkg/kubelet/cm/devicemanager/device_plugin_stub.go

```diff
@@ -82,10 +84,14 @@ func (m *Stub) Start() error {
 		return err
 	}
 
+	m.wait = make(chan interface{})
 	m.server = grpc.NewServer([]grpc.ServerOption{}...)
 	pluginapi.RegisterDevicePluginServer(m.server, m)
 
-	go m.server.Serve(sock)
+	go func() {
+		defer close(m.wait)
```

Comment:
looks like a waitgroup patern

Reply:
Yes, that's better. I hadn't used that initially because there's no need for the "group" part in WaitGroup, but it's probably still cheaper and easier to understand to just use a WaitGroup.

I've force-pushed a version of the commit with WaitGroup.